conda/exceptions.py

```diff
@@ -195,13 +199,46 @@ def __init__(self, command):
             'render',
             'skeleton',
         }
-        if command in build_commands:
+        if command in activate_commands:
+            from .base.context import context
+            builder = ["Your shell has not been properly configured to use 'conda %(command)s'."]
+            builder.append(dals("""
+            If your shell is Bash or a Bourne variant, enable conda for the current user with
```

Comment:
The message displayed below tells the user to modify their dotrc files. In some scenarios (like a shared user/service account), the dev might not want to modify the common user's dotrc file. In that scenario, one can just do 
```
. %(root_prefix)s/etc/profile.d/conda.sh
```
Should this be told to the user as part of the message before telling them that the conda function can always be made available by putting that line in their dotrc file, or should we let them infer it from the message?

Reply:
I'm still struggling with everything here, specifically the right way to communicate it all.  There are *so many* different possibilities and scenarios.  Makes it difficult to explain everything in a linear way.

I think this PR is an improvement to what we had before, and we should merge it in prep for 4.4.0.  Also, in the very near term, I need to write up documentation that goes in conda-docs explaining how activation and deactivation are fundamentally shell manipulations, how every shell is different, what it even means to be a shell manipulation rather than a "normal" conda command, etc. etc.

Once we have that URL, I'll add it to this error message.

Reply:
Fair enough, I have pushed a patch over your branch to add a TODO. Hope you don't mind! :sweat_smile:

Reply:
Don’t mind direct pushes like that at all. That’s why you have write access to my fork!